SecurityPkg/SecurityPkg.dec

```diff
@@ -637,13 +637,21 @@
   gEfiSecurityPkgTokenSpaceGuid.PcdTpm2AcpiTableLasa|0|UINT64|0x00010023
 
 
-# MU_CHANGE START TCBZ2506
+# MU_CHANGE START
 [PcdsFeatureFlag]
+  # TCBZ2506
   ## Indicates if the platform will enforce PK's to be self signed when setting the PK with none currently set
   #   TRUE  - Require PK to be self signed
   #   FALSE - Does not require PK to be self signed
   # @Prompt Enforce PK to be self signed
   gEfiMdeModulePkgTokenSpaceGuid.PcdEnforceSelfsignedPk|FALSE|BOOLEAN|0x00010024
+
+  ## Indicates if the platform will set the custom secure boot mode variable during deletion of PK

```

Comment:
Does platform code here mean the code integrating SecureBootVariableLib in SecurityPkg?

Based on the code changes, I thought this PCD means "Indicates if the platform key deletion code in this package will set the custom secure boot mode variable...".

Reply:
Thanks, this PCD should not be needed anymore.